spec/support/spree/config_helper.rb

```diff
@@ -0,0 +1,18 @@
+module Spree
+  module ConfigHelper
+    def with_spree_config(hash)
+      stored = config_slice(hash.keys)
+      Config.set(hash)
+      yield
+      Config.set(stored)
```

Comment:
`stored` is using `config_slice` to store only the keys passed into `with_spree_config`. At the end of the spec, it sets just those keys back to their original values.